easybuild/easyconfigs/m/MotionCor2/Motioncor2-wrapper.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Wrap around Motioncor2 to load the required toolchain so it can use the
+# expected version of CUDA.
+
+# Save path to the install tree.
+mc_base=$EBROOTMOTIONCOR2
+
+module unload CUDA > /dev/null 2>&1
+module add CUDA/#CUDAVER#
```

Comment:
@akesandgren As mentioned before, one issue here is that we're making assumptions about the module naming scheme here (no lowercase of the module name, etc.).

The only way to avoid that is to create an easyblock that generates this wrapper script on the fly, I think...

Reply:
Is get_software_root MNS aware then? to handle lowercase MNSes? I.e. so that get_software_root('CUDA')  (for instancs)  will actually correctly find it if it's actually called 'cUdA'

Reply:
@akesandgren `get_software_root('foo')` basically just grabs `$EBROOTFOO`, so it uses the uppercase software name, it's totally independent of the module naming scheme (you should always use the `name` as included in the easyconfig file, but casing doesn't matter)